lib/resolvers/project.js

```diff
@@ -135,6 +135,24 @@ module.exports = ({ models }) => async ({ action, data, id, meta }, transaction)
       .then(() => Project.query(transaction).patchAndFetchById(id, data));
   }
 
+  if (action === 'update-issue-date') {
+    const project = await Project.query(transaction).findById(id);
+
+    if (project.status !== 'active') {
+      throw new Error('Cannot update issue date for non-active project');
+    }
```

Comment:
This should probably be checked at the API level as well. No beef with it being here as a backstop, but it should be caught before this.

Reply:
I'll add it to the API as well